.github/workflows/ci-pytorch-tests.yml

```diff
@@ -173,6 +177,8 @@ jobs:
 
     - name: Testing PyTorch
       working-directory: tests/tests_pytorch
+      env:
```

Comment:
I suggest not doing this work in favor of https://github.com/Lightning-AI/lightning/issues/15372

Reply:
so you say that we run them on GPU nad drop this CPU version... correct?

Reply:
Yes

Reply:
ok, sounds good to me :)

Reply:
Well, thinking, are we fine with dropping this testing on all three OS? https://github.com/Lightning-AI/lightning/issues/15372#issuecomment-1295069172